vnext/package.json

```diff
@@ -62,11 +62,9 @@
     "react-native": "0.61.5"
   },
   "beachball": {
-    "defaultNpmTag": "beta",
```

Comment:
I think we also need to change the react-native windows CLI to install this version (assuming this should now be the default).
See:  [https://github.com/microsoft/react-native-windows/blob/master/packages/rnpm-plugin-windows/src/windows.js](https://github.com/microsoft/react-native-windows/blob/master/packages/rnpm-plugin-windows/src/windows.js)

Reply:
I think you're right. It passes the template as a tag and will do special prerelease logic when doing version matching IIRC. We see this cause test failures.

There's also some issue with the CLI where it doesn't correctly set exit code, so we fail in CI at a weird stage. I actually have a local fix for that. 

I'll try to get this setup and wrangle with the CLI on Monday.